src/patternfly/components/Sequence/examples/sequence-finite-steps.hbs

```diff
@@ -0,0 +1,37 @@
+{{#> sequence sequence--modifier='aria-label="Sequence"'}}
+	{{#> sequence-list sequence-list--attribute='type="1"'}}
```

Comment:
[MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul#Attributes) lists the `type` attribute as deprecated.

Reply:
thanks that was left there from when i was testing <ol>